pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md

```diff
@@ -4,18 +4,18 @@ navigationTitle: Attach Amazon EKS Cluster to Kommander
 title: Attach Amazon EKS Cluster to Kommander
 menuWeight: 7
 excerpt: Attach an existing EKS cluster to Kommander
-beta: true 
+beta: true
 enterprise: false
 ---
 
-You can attach existing Kubernetes clusters to Kommander. After attaching the cluster, you can use Kommander to [examine and manage](/ksphere/kommander/1.1.0-beta/clusters/) this cluster. The following procedure shows how to attach an existing Amazon Elastic Kubernetes Service (EKS) cluster to Kommander. 
+You can attach existing Kubernetes clusters to Kommander. After attaching the cluster, you can use Kommander to [examine and manage](/ksphere/kommander/1.1.0-beta/clusters/) this cluster. The following procedure shows how to attach an existing Amazon Elastic Kubernetes Service (EKS) cluster to Kommander.
 
-## Before you begin 
+## Before you begin
 
 This procedure requires the following items and configurations:
 
 - A fully configured and running Amazon [EKS](https://aws.amazon.com/eks/) cluster with administrative privileges.
-- Konvoy v1.5.0-beta.0 or above, [installed and configured](/ksphere/konvoy/1.5.0-beta/install/) for your Amazon EKS cluster, on your machine.
+- Konvoy v1.5.0-beta.0 or above, [installed and configured](/ksphere/konvoy/1.5/install/) for your Amazon EKS cluster, on your machine.
```

Comment:
Should we change this to just `Konvoy v1.5` ?
```suggestion
- Konvoy v1.5 or above, [installed and configured](/ksphere/konvoy/1.5/install/) for your Amazon EKS cluster, on your machine.
```

Reply:
Good catch, thank you. Will fix.